integration-scripts/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
How does this file get generated? I was under the impression that there is only 1 single `yarn.lock` when using workspaces??

Reply:
I think I created the package before adding this to the workspace, will remove now